adding support for multiple objects in a file

sstarcher opened the pull request:
Resolves #13

tomwilkie commented on kubedifflib/_diff.py:
```diff
@@ -77,22 +77,24 @@ def check_file(printer, path, kubeconfig=None):
+    expected = yaml.load_all(stream)
 
-  kube_obj = KubeObject.from_dict(expected)
+    differences = 0
```
Does all the below need indenting under the `with` block?  It seems like everything from `differences = 0` and down should be unindented by 2 spaces.

tomwilkie commented on kubedifflib/_diff.py:
```diff
@@ -77,22 +77,24 @@ def check_file(printer, path, kubeconfig=None):
+        running = kube_obj.get_from_cluster(kubeconfig=kubeconfig)
+      except subprocess.CalledProcessError, e:
+        printer.diff(path, Difference(e.output, None))
+        return 1
```
Now that this is within a loop, `return 1` is not appropriate - it should be `differences += 1` and `continue`

sstarcher commented:
I fixed the return 1 issue.  I moved it under the open block as load_all is lazy and does not actually load the file until you attempt to access it and will throw a file closed exception.

tomwilkie commented:
> I fixed the return 1 issue. I moved it under the open block as load_all is lazy and does not actually load the file until you attempt to access it and will throw a file closed exception.

Ah I see.  Thanks!

tomwilkie commented:
> I fixed the return 1 issue. I moved it under the open block as load_all is lazy and does not actually load the file until you attempt to access it and will throw a file closed exception.

Ah I see.  Thanks!
